pyveg/src/coordinate_utils.py

```diff
@@ -8,10 +8,29 @@
 import requests
 
 
-def get_region_string(coords, region_size):
+def get_coords(bounds):
```

Comment:
I don't think that this function is required any more.

Reply:
I'm leaving these functions there for now in the utils, just in case we want to use them later. But we should remove them once the pipeline is fixed.